mass/adapters/inbound/event_sub.py

```diff
@@ -0,0 +1,154 @@
+# Copyright 2021 - 2023 Universität Tübingen, DKFZ, EMBL, and Universität zu Köln
+# for the German Human Genome-Phenome Archive (GHGA)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Event subscriber details for searchable resource events"""
+import logging
+
+import ghga_event_schemas.pydantic_ as event_schemas
+from ghga_event_schemas.validation import (
+    EventSchemaValidationError,
+    get_validated_payload,
+)
+from hexkit.custom_types import Ascii, JsonObject
+from hexkit.protocols.eventsub import EventSubscriberProtocol
+from pydantic import BaseSettings, Field
+
+from mass.core.models import Resource
+from mass.ports.inbound.query_handler import (
+    ClassNotConfiguredError,
+    DeletionFailedError,
+    QueryHandlerPort,
+)
+
+CLASS_NOT_CONFIGURED_LOG_MSG = "Class with name %s not configured."
+
+DELETION_FAILED_LOG_MSG = (
+    "Failed to delete resource with ID '%s' because "
+    + "no match was found in the database."
+)
+SCHEMA_VALIDATION_ERROR_LOG_MSG = "Failed to validate event schema for '%s'"
+
+logging.basicConfig()
```

Comment:
You're right. I'd gotten in a habit of using that during debugging in order to log lower level statements that I forgot it wouldn't be required here. It will get called automatically if it hasn't been called yet, and the default level works for us for now. [15f8893](https://github.com/ghga-de/mass/pull/12/commits/15f88933b84eb580d8c7c1f01317f748b42b80aa)